sgv2-restapi/src/main/java/io/stargate/sgv2/restsvc/resources/Sgv2RowsResourceImpl.java

```diff
@@ -393,15 +365,15 @@ protected String buildGetRowsByPKCQL(
         .from(keyspaceName, tableName)
         .where(whereConditions)
         .orderBy(sortOrder)
+        .parameters(parametersForPageSizeAndState(pageSizeParam, pageStateParam))
         .build();
   }
 
-  private String buildDeleteRowsByPKCQL(
+  private QueryOuterClass.Query buildDeleteRowsByPKCQL(
```

Comment:
Similarly here "ByPKCQL" -> "ByPKQuery"

Reply:
Two more methods as well, I've renamed all of them.